packages/perseus-editor/src/iframe-content-renderer.tsx

```diff
@@ -14,6 +14,48 @@
 import {Log} from "@khanacademy/perseus";
 import * as React from "react";
 
+import type {
+    APIOptions,
+    DeviceType,
+    PerseusItem,
+    PerseusRenderer,
+} from "@khanacademy/perseus";
+import type {LinterContextProps} from "@khanacademy/perseus-linter";
+
+type ArticleData = {
+    apiOptions: APIOptions;
+    json: Partial<PerseusRenderer>;
+    useNewStyles: boolean;
+    linterContext: LinterContextProps;
+    legacyPerseusLint: ReadonlyArray<string>;
+};
+
+export type SendNewData =
```

Comment:
Could we call this type something like `NewDataMessage`? The verb `Send` makes it sound like a function type.

Reply:
Yes! `SendNewData` just didn't feel right, but I couldn't think of a better alternative. :) Thanks!